.gitignore

```diff
@@ -63,3 +63,5 @@ fastlane/report.xml
 fastlane/screenshots
 APIKEY
 
+# WhirlyGlobe-Maply and its associated submodules are checked out here
+Submodules/WhirlyGlobe
```

Comment:
Probably don't need to ignore this. Git submodules will understand it well enough...